pyproject.toml

```diff
@@ -49,6 +49,7 @@ PyTango = { version = "^9.3.6"}
 python-ldap = "^3.4.0"
 requests = "^2.31.0"
 colorama = "^0.4.6"
+pyicat-plus = ">=0.4.0"
```

Comment:
```suggestion
pyicat-plus = {version ">=0.4.0", , optional=true}
```

Reply:
I think, that should work, then we can do `poetry install -E pyicat-plus` or `pip install mxubecore [pyicat-plus]`

Reply:
Actually I would not be against some site specific section either.